docs/Spatial.md

```diff
@@ -1,6 +1,99 @@
+
+##Data
+ 
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|This is an identifier, Phidget GPS will always return 'PhidgetSpatial'|
+|accelerationMin|number|no|This returns the maximum negative acceleration that the desired axis will report.|
+|accelerationMax|number|no|This returns the maximum acceleration that the desired axis will report.|
+|angularRateMax|number|no|This returns the maximum angular momentum that the desired axis will report.|
+|angularRateMin|number|no|This returns the maximum angular momentum that the desired axis will report. |
+|dataRate|number|yes|This controls and reports current data rate at which data is being handled. In depth guide [here](http://www.phidgets.com/docs/1044_User_Guide) |
+|dataRateMax|number|no|This returns the maximum supported data rate.|
+|dataRateMin|number|no|is returns the minimum supported data rate.|
+|compassAxisCount|number|no|This returns the number of axes the PhidgetSpatial can measure magnetic field strength on. |
+|gyroAxisCount|number|no|This returns the number of axes the PhidgetSpatial can measure angular momentum on. |
+|accelerationAxisCount|number|no|This returns the number of axes the PhidgetSpatial can measure acceleration on. |
+|magneticFieldMax|number|no|This returns the maximum magnetic field that this axis will report.|
+|magneticFieldMin|number|no|This returns the maximum negative magnetic field that this axis will report.|
+|observeAcceleration|listener|no|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) the PhidgetSpacial for any acceleration changes and reports their activity.|
+|observeGyro|listener|no|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) the PhidgetSpacial for any changes in angular momentum and reports their activity.|
+|observeMagneticField|listener|no|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) the PhidgetSpacial for any changes in magnetic fields and reports their activity.|
+|acceleration|array|no|This contains three directions of acceleration with [user defined coordinates](http://www.phidgets.com/docs/1044_User_Guide) in __g__.|
+|acceleration|array|no|This contains three directions of acceleration with [user defined coordinates](http://www.phidgets.com/docs/1044_User_Guide) in __g__.|
+|gyro|array|no|This contains three directions of angular momentum with [user defined coordinates](http://www.phidgets.com/docs/1044_User_Guide) in __<sup>o</sup>/s__.|
+|magneticField|array|no|This contains three directions of magnetic field with [user defined coordinates](http://www.phidgets.com/docs/1044_User_Guide) in __G (Gauss)__.|
+|dataRate|number|yes|This controls and reports the rate at which data is acquired.|
```

Comment:
should just read

` the number of times data is acquired per second. ` 

you specified it is writable that is enough for saying it is settable. Otherwise there is no reason to have a writable column.